sdk/core/Azure.Core.TestFramework/src/RecordedTestAttribute.cs

```diff
@@ -39,6 +39,12 @@ public FallbackCommand(TestCommand innerCommand) : base(innerCommand)
             { }
             public override TestResult Execute(TestExecutionContext context)
             {
+                // Lift the global timeout setting if we are not in Playback mode
+                var recordedTest = context.TestObject as RecordedTestBase;
+                if (recordedTest?.Mode != RecordedTestMode.Playback)
+                {
+                    context.TestCaseTimeout = 0; // set the default timeout;
```

Comment:
experimental at this point. If this works well, we may want to walk up the class hierarchy to find the lowest level Timeout attribute below assembly level set and use that, if one exists.